index.js

```diff
@@ -142,9 +141,6 @@ module.exports = {
 
         'space-infix-ops': 'error',
         'no-case-declarations': 0,
-        /* eslint-disable-next-line max-len */
```

Comment:
Does comment really need it? It should be exempted in max-len, no?

Reply:
Hmm? It is removed, its highlighted in red, no?

Reply:
Ah, right, got confused lol